rayon-core/src/lib.rs

```diff
@@ -144,7 +144,7 @@ impl Configuration {
         if self.num_threads > 0 {
             self.num_threads
         } else {
-            match env::var("RAYON_RS_NUM_CPUS").ok().and_then(|s| usize::from_str(&s).ok()) {
+            match env::var("RAYON_RS_NUM_THREADS").ok().and_then(|s| usize::from_str(&s).ok()) {
```

Comment:
Certainly give priority to the new and documented name.

This is in rayon-core, which *is* 1.0 already.  Environment variables are a gray area in an API, but since it was documented, and it's easy to keep the compatible fallback, I think we should.